pkg/bigquerysql/sql.go

```diff
@@ -0,0 +1,125 @@
+package bigquerysql
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tiflow/pkg/sink/cloudstorage"
+)
+
+var (
+	CDC_FLAG_COLUMN_NAME       = "tidb2dw_flag"
+	CDC_TABLENAME_COLUMN_NAME  = "tidb2dw_tablename"
+	CDC_SCHEMANAME_COLUMN_NAME = "tidb2dw_schemaname"
+	CDC_TIMESTAMP_COLUMN_NAME  = "tidb2dw_timestamp"
```

Comment:
<img width="791" alt="image" src="https://github.com/pingcap-inc/tidb2dw/assets/60744015/969938bd-21ea-4f98-842c-ea1c7606742c">

rename to `CDC_COMMIT_TS_COLUMN_NAME` ?